src/main/resources/openapi/sqs/print-api-sqs-messaging.yaml

```diff
@@ -170,6 +188,10 @@ components:
         postcode:
           type: string
           maxLength: 10
+        uprn:
```

Comment:
`uprn` was missing from the original `Address` type, even though it was described in the examples

Reply:
We don't send this to the printers so it has no use in the print api (yet). Feel free to store it too.

Reply:
Yeah, thats an interesting one actually. For printing we wont use the UPRN, the printer would never use it, its not printed as part of the address. So there's a strong argument that says we dont need it / shouldnt accept it / not part of the message type.
But, the address description and example suggests we should include it as a field:
```
    Address:
      title: Address
      description: Address format used throughout gov.uk voter services.
      type: object
      x-examples:
        Minimum data:
          street: Street 1
          postcode: PC1 2FB
        Normal Address:
          street: East Lodge
          property: Balruddery
          locality: Invergowrie
          town: Dundee
          area: Angus
          postcode: DD25LF
          uprn: '117095813'
    ....
```
especially the description which says "Address format used throughout gov.uk voter services."

Given that its the same address format used in all the services I think it should include the field, but I take your point that we'll do nothing with it, so we'll consider how/if we store it in the DB when we get to that ticket 👍